Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -986,6 +986,13 @@ void CGameListCtrl::OnRightClick(wxMouseEvent& event)
       {
         popupMenu.Append(IDM_OPEN_SAVE_FOLDER, _("Open Wii &save folder"));
         popupMenu.Append(IDM_EXPORT_SAVE, _("Export Wii save (Experimental)"));
+
+        // We should not allow the user to mess with the save folder or export saves while
+        // emulation is running, because this could result in the exported save being in
+        // an inconsistent state; the emulated software can do *anything* to its data directory,
+        // and we definitely do not want the user to touch anything in there if it's running.
+        for (const int index : {IDM_OPEN_SAVE_FOLDER, IDM_EXPORT_SAVE})
+          popupMenu.FindItem(index)->Enable(!Core::IsRunning() || !SConfig::GetInstance().bWii);
```

Comment:
You don't need to re-find these. `Append` returns a pointer to the added menu item, so this can just be

```cpp
auto* const open_save_folder_item = popupMenu.Append(IDM_OPEN_SAVE_FOLDER, _("Open Wii &save folder"));
auto* const export_save_item = popupMenu.Append(IDM_EXPORT_SAVE, _("Export Wii save (Experimental)"));

for (auto* menu_item : {open_save_folder_item, export_save_item})
  menu_item->Enable(!Core::IsRunning() || !SConfig::GetInstance().bWii);
```